environments/hca-prod/terraform/buckets.tf

```diff
@@ -22,7 +22,7 @@ resource google_storage_bucket_iam_member ebi_sa_bucket_iam {
   # When the storage.admin role is applied to an individual bucket,
   # the control applies only to the specified bucket and objects within
   # the bucket: https://cloud.google.com/storage/docs/access-control/iam-roles
-  for_each = toset(["ebi-staging-writer@broad-dsp-monster-hca-dev.iam.gserviceaccount.com"])
+  for_each = toset([])
```

Comment:
🔧 I think the entire `ebi_sa_bucket_iam` definition becomes a noop and should be removed